core/src/test/java/org/elasticsearch/index/query/PercolatorQueryBuilderTests.java

```diff
@@ -75,6 +79,16 @@ private PercolatorQueryBuilder doCreateTestQueryBuilder(boolean indexedDocument)
         }
     }
 
+    /**
```

Comment:
@cbuescher Thx, now I understand it is about the reordering of the fields inside the document itself. Maybe make sure that the comment emphasizes this?